src/main/java/org/spongepowered/api/util/event/factory/ClassGenerator.java

```diff
@@ -81,6 +84,7 @@
 
     private final PropertySearchStrategy propertySearch = new AccessorFirstStrategy();
     private NullPolicy nullPolicy = NullPolicy.DISABLE_PRECONDITIONS;
+    private final List<String> primitivePropertyExceptions = Collections.singletonList("cancelled");
```

Comment:
Do you use the List here so new exceptions can be added easily? If so, it should probably use `ImmutableList.of` instead so you don't need to switch the collection type and can add new elements directly to the method call